openapi/v2/system-agent.yaml

```diff
@@ -371,19 +369,6 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HealthResponse'
```

Comment:
In the example, please state "healthy", not just string

Reply:
An example is provided: `edgex-core-data: healthy`